.github/ISSUE_TEMPLATE/website-request.md

```diff
@@ -7,31 +7,31 @@ assignees: ''
 
 ---
 
-> **This request is expected to be doable in ≤1 sprint.  It would be valuable even if nothing else ships.**
+> **This request is expected to be doable in ≤1 sprint. It would be valuable even if nothing else ships.**
 > 
 > It will be reviewed by the acting PM for the #g-website product group, and then hopefully [prioritized, drafted, and implemented](https://fleetdm.com/handbook/company/development-groups#making-changes).
 
 ## Goal
 
-<!-- Describe the desired outcome -->
+<!-- Describe the desired outcome.-->
 
 | User story  |
 |:---------------------------------------------------------------------------|
 | As a _________________________________________,
 | I want to _________________________________________
 | so that I can _________________________________________.
 
+>For help creating a user story, see ["Writing a good user story"](https://fleetdm.com/handbook/company/development-groups#writing-a-good-user-story) in the website handbook.
+
 ## Changes
 
-This issue's estimation includes completing:
+This issue's estimation includes completing the following:
 - [ ] UI changes: TODO
 - [ ] QA complete?
 
 ## QA
 
-<!--
-Make sure that someone verified each step outlined below before the associated PR is merged.
---> 
+>The testing steps outlined below must be verified before the associated PR is merged. See ["Quality"](https://fleetdm.com/handbook/marketing/website-handbook#quality) in the website handbook for help.
```

Comment:
@mike-j-thomas @eashaw I think we could probably just cut the testing steps section, to err on the side of making it easier to submit requests properly.  That said, it's kind of a separate thing than what this PR is about